ELM_CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Elm Changelog
 
+## [15.0.0]
+
+- Changed `filter` on `ColumnDef` to `Maybe FilterType`, which defaults to `Nothing`. This differs from the previous `Filter` default, which would be `Just SetFilter`. `Nothing` indicates the use of the default filter and filterValueFormatter determined by the column renderer. If a `Just` value is specified, the default value is overridden. So if you don't want to enforce a filter on the column, you need to set it to `Just NoFilter` and not `Nothing`.
+
```

Comment:
I don't understand the difference between Nothing and Just NoFilter.

Reply:
Nothing == default formatter will be used (based on the Renderer)
Just NoFilter = NoFilter will be set (this overrides the default formatter)

Reply:
Sounds good. We now have a `DefaultFilter` and a `NoFilter` variant. It is no more a `Maybe`